src/components/Chip/Chip.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { ButtonHTMLAttributes, ReactNode } from 'react'
+import styled from '@emotion/styled'
+import { css } from '@emotion/react'
+import {
+  space,
+  color,
+  device,
+  lineHeight,
+  fontSize,
+  fontWeight,
+} from '../../theme'
+
+const Content = styled.span`
+  white-space: nowrap;
+  overflow: hidden;
+  text-overflow: ellipsis;
+  pointer-events: auto;
+`
+
+interface ContainerProps {
+  active: boolean
+  compact: boolean
+  hasStartAdornment: boolean
+}
+
+const Container = styled.span<ContainerProps>`
+  --_accentColor: ${color.inactive};
+  --_backgroundColor: ${color.background};
```

Comment:
I get why you're doing this but personally, I don't like it. I would rather call this `ChipAccentColor` instead of `_`. If you don't know that we have those variables in our theme it can be really weird to see a `_` before.

Reply:
It's a convention to start local variables with `_`.

Reply:
@RobVermeer Link to convention?

Reply:
https://github.com/argyleink/open-props/blob/main/src/extra/buttons.css
See how they do it in `open-props`, created by one of the best CSSers Adam Argyle 🤓

Reply:
They also seem to do both `var(--_accent, var(--link));` so then it's a bit confusing why there is a difference. I find the `_` less easy to the eye but i have no strong opinion about it. We just need to stick to our convention..